File: java-gradle/plugins/kafka/src/gatling/java/{{ .PackagePath }}/{{ .NameWord }}/cases/KafkaActions.java
package {{ .Package }}.{{ .NameWord }}.cases;

import io.gatling.javaapi.core.ChainBuilder;
import org.galaxio.gatling.kafka.javaapi.KafkaDsl;

import static io.gatling.javaapi.core.CoreDsl.exec;
import static org.galaxio.gatling.javaapi.SimulationConfig.getStringParam;

public final class KafkaActions {

    public static final ChainBuilder sendMessage =
            exec(KafkaDsl.kafka("Kafka publish").send("myMessage", "Hello!"));

    private KafkaActions() {
    }
}

File: kotlin-gradle/plugins/kafka/src/gatling/kotlin/{{ .PackagePath }}/{{ .NameWord }}/cases/KafkaActions.kt
package {{ .Package }}.{{ .NameWord }}.cases

import io.gatling.javaapi.core.ChainBuilder
import io.gatling.javaapi.core.CoreDsl.exec
import org.galaxio.gatling.kafka.javaapi.KafkaDsl
import org.galaxio.gatling.javaapi.SimulationConfig.getStringParam

object KafkaActions {
    val sendMessage: ChainBuilder = exec(
        KafkaDsl.kafka("Kafka publish").send("myMessage", "Hello!")
    )
}
